<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="yuri-PC" name="Broadcaster - multiple UDP stream creation diff source" tests="1" failures="1" timestamp="25 Mar 2018 15:14:28 GMT" time="46.656" errors="0">
  <testcase name="broadcasterMultipleUdpCreation" time="41.215" classname="com.zixi.load.testing.BroadcasterMultipleUdpManualTest">
    <failure type="java.util.concurrent.ExecutionException" message="com.zixi.tools.BroadcasterInitialSecuredLogin$1: Loggin parameters are empty.">
      <![CDATA[java.util.concurrent.ExecutionException: com.zixi.tools.BroadcasterInitialSecuredLogin$1: Loggin parameters are empty.
	at java.util.concurrent.FutureTask.report(Unknown Source)
	at java.util.concurrent.FutureTask.get(Unknown Source)
	at com.zixi.threads.ZthreadPool.zexecuteUdpManual(ZthreadPool.java:373)
	at com.zixi.load.drivers.BroadcasterMultipleUdpDriver.testIMPLManual(BroadcasterMultipleUdpDriver.java:34)
	at com.zixi.load.testing.BroadcasterMultipleUdpManualTest.broadcasterMultipleUdpCreation(BroadcasterMultipleUdpManualTest.java:50)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:85)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:659)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:845)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1153)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:108)
	at org.testng.TestRunner.privateRun(TestRunner.java:771)
	at org.testng.TestRunner.run(TestRunner.java:621)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:357)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:352)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:310)
	at org.testng.SuiteRunner.run(SuiteRunner.java:259)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1199)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1124)
	at org.testng.TestNG.run(TestNG.java:1032)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:111)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:204)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:175)
Caused by: com.zixi.tools.BroadcasterInitialSecuredLogin$1: Loggin parameters are empty.
	at com.zixi.tools.BroadcasterInitialSecuredLogin.sendGet(BroadcasterInitialSecuredLogin.java:81)
	at com.zixi.drivers.drivers.BroadcasterSingleUdpInCreationDriver.testIMPL(BroadcasterSingleUdpInCreationDriver.java:19)
	at com.zixi.threads.ZthreadPool$5.call(ZthreadPool.java:329)
	at com.zixi.threads.ZthreadPool$5.call(ZthreadPool.java:1)
	at java.util.concurrent.FutureTask.run(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
	at java.lang.Thread.run(Unknown Source)
]]>
    </failure>
  </testcase> <!-- broadcasterMultipleUdpCreation -->
</testsuite> <!-- Broadcaster - multiple UDP stream creation diff source -->
